src/frontends/paddle/src/op/unstack.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs unstack(const NodeContext& node) {
+    auto data = node.get_input("X");
+    auto dim = node.get_attribute<int32_t>("axis");
+    if (dim < 0) {
+        dim = dim + data.get_partial_shape().rank().get_length();
```

Comment:
Can we remove this line, because split op can accept negative axis: https://github.com/openvinotoolkit/openvino/blob/master/docs/articles_en/documentation/openvino_ir/operation_sets/operations_specifications/movement/Split_1.md

Reply:
done

Reply:
have you validated it again, and could you help to update your uni-test screenshot ?

Reply:
sorry, missed the notification email, we should keep these code because index access of [`shape`](https://github.com/openvinotoolkit/openvino/pull/20080/files#diff-18009a584c0413af6acc68760c8994ec6ba885713de085e8d00323e4046f750dR20) is bounded within [0, abs(dim)-1]